pcli/src/box_grpc_svc.rs

```diff
@@ -23,23 +23,25 @@ pub(crate) async fn connect(ep: Endpoint) -> anyhow::Result<BoxGrpcService> {
         .map_response(|rsp: grpc::Response<transport::Body>| rsp.map(box_rsp_body))
         .map_err(BoxError::from)
         .service(conn);
-    Ok(UnsyncBoxService::new(svc))
+    Ok(BoxCloneService::new(svc))
 }
 
 /// Constructs a [`BoxGrpcService`] by erasing the type of an `S`-typed local
 /// (in-process) service instance.
 pub(crate) fn local<S, B>(svc: S) -> BoxGrpcService
 where
-    S: Service<grpc::Request<ReqBody>, Response = grpc::Response<B>> + 'static,
-    BoxError: From<S::Error> + From<B::Error>,
+    S: Service<grpc::Request<ReqBody>, Response = grpc::Response<B>>,
+    S: Clone + Send + Sync + 'static,
```

Comment:
What are the implications of requiring that `S: Sync`? Aren't most tower middlewares not `Sync`?